src/promptflow/promptflow/_cli/_pf/entry.py

```diff
@@ -89,6 +95,16 @@ def entry(argv):
             init_finish_time - start_time,
             invoke_finish_time - init_finish_time,
         )
+        activity_name = f"pf.{args.action}s.{args.sub_action}.e2e"
```

Comment:
no need to add s here

Reply:
ok, but it seems that our original command for management had's'.